### PR TITLE
feat: add Cardano support to getAccountInfo method

### PR DESCRIPTION
**Pull request by mroz22**
- [x] depends on https://github.com/trezor/trezor-firmware/pull/2016

### CODE REVIEW

**Comment by vladimirvolek** [comment]
> todos:
> 
> * [x]  add cardano testnet here https://github.com/trezor/trezor-firmware/blob/30a77a76bc41296a32269694a666fadfc79d89ec/common/defs/misc/misc.json#L89

added in https://github.com/trezor/trezor-firmware/pull/2016

File: src/data/coins.json [hunk 1]
```diff
@@ -2735,6 +2735,25 @@
...
+                "type": "blockfrost",
+                "url": [
+                    "wss://trezor-cardano-testnet.blockfrost.io"
+                ]
+            },
+            "curve": "ed25519",
+            "decimals": 6,
+            "name": "Cardano Testnet",
+            "shortcut": "tADA",
+            "slip44": 1815,
```
Comment (mroz22) [inline src/data/coins.json:2749]: use whatever will be result in firmware repo

File: src/js/utils/accountUtils.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ export const getAccountAddressN = (
     if (coinInfo.type === 'bitcoin') {
         return [toHardened(options.purpose), toHardened(options.coinType), toHardened(index)];
     }
+    // see: https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/CIP-1852.md
+    if (coinInfo.shortcut === 'ADA' || coinInfo.shortcut === 'tADA') {
+        return [toHardened(1852), toHardened(coinInfo.slip44), toHardened(index)];
```
Comment (mroz22) [inline src/js/utils/accountUtils.js]: 1852 ? 
coinInfo.slip44 vs options.coinType

**Review by marekrjpolak — APPROVED** [review]
LGTM

**Comment by mroz22** [comment]
:shipit:

**Comment by vladimirvolek** [comment]
🥳    🥳  🥳   🥳    🥳  🥳   🥳 🥳🥳 🥳 🥳 🥳

**Pull request merged**